the-basics/building-thunderbird/linux-build-prerequisites.md

```diff
@@ -6,3 +6,58 @@ description: >-
 
 # Linux Build Prerequisites
 
+This guide assumes you already followed the [Getting Started](../untitled.md) documentation and you already downloaded `mozilla-central` and `comm-central` source code.
+
+## 64-bit version
+
+You need a 64 bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
+
+```text
+uname -m
+```
+
+if this command returns `x86_64` you can proceed.
+
+## Python 2.7
+
+You’ll need `Python 2.7` installed, `Python 3.x` is currently not supported. 
+
+You can check with `python --version` to see if you have it already. If not, you can install it with your distribution’s package manager. Make sure your system is up to date!
+
+## 30 GB of free space
+
+The Thunderbird build will take up to 30 GB of disk space in order to complete. Be sure to have enough free space and a fast internet connection to avoid interruptions.
+
+## Bootstrap your system
+
+Access the location where you downloaded the `mozilla-central` source code, most likely `source/` and trigger this command:
+
+```text
+./mach bootstrap
+```
+
```

Comment:
Should we put something here that tells the user what to select from this list?

Please choose the version of Firefox you want to build:
1. Firefox for Desktop Artifact Mode
2. Firefox for Desktop
3. GeckoView/Firefox for Android Artifact Mode
4. GeckoView/Firefox for Android

I selected number 2. I assume that is the only one that works?

Reply:
Mh, I don't think it's necessary as those choices are thoroughly explained in the terminal once the `bootstrap` command is running.
I personally selected option 1 to have those C artifacts already built and speed up my first building process.

Reply:
I'll pull it in and we can think on it more.

Reply:
Artifact Mode doesn't work for Thunderbird (though Geoff is working on fixing that).